zepid/base.py

```diff
@@ -743,7 +892,65 @@ def summary(self, decimal=3):
         print('Missing T:   ', self._missing_t)
         print('======================================================================')
 
+    def plot(self, measure='incidence_rate_difference', color='k', center=0):
+        """Plot the incidence rate differences or the incidence rates along with their corresponding confidence
+        intervals. This option is an alternative to summary(), which displays results in a table format.
+
+        Parameters
+        ----------
+        measure : str, optional
+            Whether to display incidence rate ratios or incidence rates. Default is to display the incidence rate
+            differences. Options are;
+            * 'incidence_rate_difference'  : display incidence rate differences
+            * 'incidence_rate'             : display incidence rates
+        color : str, optional
+            Color to display points and confidence limits. Allows any valid matplotlib colors
+        center : str, optional
+            Sets a reference line. For the incidence rate difference, the reference line defaults to 0. For incidence
+            rates, no reference line is displayed.
+
+        Returns
+        -------
+        matplotlib axes
+        """
+        if measure == 'incidence_rate_difference':
+            ax = _plotter(estimate=self.results['IncRateDiff'], lcl=self.results['IRD_LCL'],
+                          ucl=self.results['IRR_UCL'], labels=self.results.index,
+                          center=center, color=color)
+            ax.set_title('Incidence Rate Difference')
+        elif measure == 'incidence_rate':
+            ax = _plotter(estimate=self.results['IncRate'], lcl=self.results['IncRate_LCL'],
+                          ucl=self.results['IncRate_UCL'], labels=self.results.index,
+                          center=np.nan, color=color)
+            ax.set_title('Incidence Rate')
+            ax.set_xlim([0, 1])
+        else:
+            raise ValueError('Must specify either "incidence_rate_difference" or "incidence_rate" for plots')
+        return ax
+
+
+def _plotter(estimate, lcl, ucl, labels, center=0, color='k'):
```

Comment:
ppl will want to override your plotting settings (like `s`, `marker`, etc.). Unless you want to be strict & opinionated (which is fine), a suggestion is to add `**plot_kwargs` to the function here. Ex:
```
def plot(self, measure='incidence_rate_difference', center=0, **plot_kwargs):
    ...
    ax = _plotter(..., **plot_kwargs)

def _plotter(estimate, lcl, ucl, labels, center=0, **plotting_kwargs):
   ...
   # here you could also set defaults for plotting_kwargs, which will not overwrite if provided:
   plotting_kwargs.set_default('s', 100)
   plotting_kwargs.set_default('edgecolors', None)
   ...
   ax.scatter(estimate, ypoints, **plotting_kwargs)
```

Now users can customize their scatter plots

Reply:
That's a better way. Makes it easier for me as well. Thanks for the suggestion!